projects/whydense/README.md

```diff
@@ -6,4 +6,6 @@ Link to paper: https://arxiv.org/abs/1903.11257.
 
 This repository contains code which was originally used in the paper "How Can We Be So Dense? The Benefits of Using Highly Sparse Representations" (https://arxiv.org/abs/1903.11257).
 
+The code relies on nupic.torch (link to: https://github.com/numenta/nupic.torch - Connect to preview ). nupic.torch contains easy to use pytorch implementations of the two types of sparsity used in the paper: the k-winner-take-all sparse activation function and sparse weights.
```

Comment:
The "Connect to preview" string is part of the JIRA UI not the text itself. 
I believe the intention on the JIRA is to create a markdown link to "https://github.com/numenta/nupic.torch" under the words "nupic.torch". Something like this:
```
...
The code relies on [nupic.torch](https://github.com/numenta/nupic.torch).
```

Reply:
changed, did not realize the hyperlink formatting error. thanks for pointing this out